### DISCUSSION TITLE
Run on silent mode.

### DISCUSSION THREAD

**Discussion by KNATE3123** [Raphire/Win11Debloat#165] [Q&A]
Hi all, 

can anyone assist as i notice there is silent mode but when i try running it seems im getting the error below.
S:\Win11Debloat-Master>PowerShell -ExecutionPolicy Bypass -Command "& {Start-Process PowerShell -ArgumentList '-NoProfile -ExecutionPolicy Bypass -File ""S:\Win11Debloat-Master\Win11Debloat.ps1""'  -Silent -RunDefaults -Verb RunAs}"
Start-Process : A parameter cannot be found that matches parameter name 'Silent'.
At line:1 char:127
+ ... ypass -File "S:\Win11Debloat-Master\Win11Debloat.ps1' -Silent -RunDef ...
+                                                           ~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-Process], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.StartProcessCommand

can anyone help me??

**Comment by Raphire** [comment]
Heya,

I see you edited the run file to include those parameters, but it doesn't allow you to pass those parameters like this. If possible, I'd suggest using either the `quick method` or `advanced method` [outlined here](https://github.com/Raphire/Win11Debloat?tab=readme-ov-file#usage) to run the script instead of editing the `run.bat`. 

An example for the quick method powershell command for running the script with `RunDefault` and `Silent` parameters: 
```powershell
& ([scriptblock]::Create((irm "https://win11debloat.raphi.re/"))) -RunDefaults -Silent
```
